composer.json

```diff
@@ -91,5 +104,23 @@
         }
     },
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+    "scripts": {
+        "post-install-cmd": [
+            "composer run-script post-update-cmd"
+        ],
+        "pre-update-cmd": [
+            "@putenv COMPOSER_MEMORY_LIMIT=2G"
+        ],
+        "post-update-cmd": [
+            "composer run-script remove-git-submodules"
+        ],
+        "acms:install": [
+            "bash ./docroot/profiles/contrib/acquia_cms/install-acms"
+        ],
+        "nuke": [
+            "rm -rf docroot vendor composer.lock"
+        ],
+        "remove-git-submodules": "find . -mindepth 2 -type d -name .git | xargs rm -rf"
```

Comment:
Nit, these aren't submodules, they are Git repos that _become_ submodules if committed to a parent Git repo. This seems like the wrong way to prevent that. Any Git repos installed by Composer should be gitignored instead.

Reply:
I'm also not super comfortable running `rm -rf` on a customer's machine in any context. For instance, if someone has a Composer path repo configured globally, this command could traverse symlinks and delete directories outside of the current repo. This is a footgun that I've personally pulled the trigger on in the past, and would prefer not to repeat.

Reply:
Now that you point this out, I don't think this should be necessary now. This would have been relevant when the acms repo was private, but I don't see that composer would be installing any git repos now.